etc/config/typescript.defaults.properties

```diff
@@ -0,0 +1,18 @@
+# Default settings for TypeScript Compiler
+compilers=&tsc
+compilerType=typescript
+versionFlag=--version
+
+group.tsc.compilers=tsc_jit_gc:tsc_jit_nogc
+compiler.tsc_jit_gc.exe=/opt/compiler-explorer/tsc/bin/tsc
```

Comment:
is there maybe a better "default" path? There may not be, as if there's not an obvious path it'll get installed to. This is fine if not